src/dashboard/frontend/src/index.css

```
@@ -194,7 +279,7 @@ body::after {
 }
 
 .terminal-output {
-  font-family: 'SF Mono', 'SFMono-Regular', Consolas, 'Liberation Mono', monospace;
+  font-family: var(--font-mono), 'SF Mono', 'SFMono-Regular', Consolas, 'Liberation Mono', monospace;
```

Stylelint (17.14.1)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)